com.stremio.Service.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id": "com.stremio.Service",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "/app/bin/stremio-service/stremio-service",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.kde.StatusNotifierWatcher"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "modules": [{
+        "name": "service",
+        "buildsystem": "simple",
+        "build-options": {
+            "env": {
+                "CARGO_HOME": "/run/build/service/cargo"
+            }
+        },
+        "build-commands": [
+            "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+            "cargo --offline build --release --features=bundled,offline-build --verbose"
+        ],
+        "post-install": [
+            "install -Dm755 server.js -t /app/bin/stremio-service/",
+            "install -Dm755 ./target/release/stremio-service -t /app/bin/stremio-service/",
+            "install -Dm755 ./resources/bin/linux/node -t /app/bin/stremio-service/",
+            "install -Dm755 ./resources/bin/linux/ffmpeg -t /app/bin/stremio-service/",
+            "install -Dm755 ./resources/bin/linux/ffprobe -t /app/bin/stremio-service/",
+            "install -Dm644 ./resources/com.stremio.service.svg /app/share/icons/hicolor/scalable/apps/com.stremio.Service.svg",
+            "install -Dm644 ./flatpak/com.stremio.Service.desktop /app/share/applications/com.stremio.Service.desktop",
+            "install -Dm644 ./flatpak/com.stremio.Service.appdata.xml /app/share/metainfo/com.stremio.Service.appdata.xml"
+        ],
+        "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/Stremio/stremio-service.git",
+                "tag": "74143f710c3a4b668aee3a08da586e245405757f"
+            },
+            "./cargo-sources.json",
+            "./server-source.json"
```

Comment:
there is strictly no reason to have a separe "server-source.json"

Reply:
It's just easier for us to do that at the moment as we use a build script on our repo to generate this file, there is your reason